CMakeLists.txt

```diff
@@ -45,7 +45,7 @@ set(GOOGLE_CLOUD_CPP_GOOGLEAPIS_SHA256
     "6b8a9b2bcb4476e9a5a9872869996f0d639c8d5df76dd8a893e79201f211b1cf")
 
 include(ExternalProject)
-externalproject_add(googleapis_download
+ExternalProject_Add(googleapis_download
```

Comment:
This repo already has the same `.cmake-format.py`. I needed to update cmake-format to the version 0.5.2 because of the missing feature in 0.5.1, so maybe it's caused by the version difference?

Reply:
Ah, yes, that makes sense.

Reply:
FYI.  I like 0.5.2's formatting better, thanks for doing that.